JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/derivation/DerivationSparql.java

```diff
@@ -190,7 +203,7 @@ public DerivationSparql(StoreClientInterface storeClient, String derivationInsta
 	 * @return
 	 */
 	List<String> unifiedBulkCreateDerivations(List<List<String>> entitiesList, List<String> agentIRIList,
-			List<String> agentURLList, List<List<String>> inputsList, List<Iri> derivationTypeList, List<Boolean> forAsyncUpdateFlagList) {
+			List<String> agentURLList, List<List<String>> inputsList, List<String> derivationTypeList, List<Boolean> forAsyncUpdateFlagList) {
```

Comment:
This is not specific to this PR, but I don't like passing the derivation types as strings that can get wrong easily. They should be either enums or defined as part of the Derivation class